e2e/README.md

```diff
@@ -0,0 +1,6 @@
+## End-to-End tests
+
+End-to-End tests require running backend and frontend. To run tests type `npm start` and choose Specification file with
```

Comment:
Can you give a direct example on how to choose `Specification` file and where it is located?